packages/vite/src/ember.ts

```diff
@@ -112,3 +95,31 @@ export function ember() {
 function shouldBuildTests(mode: string) {
   return mode !== 'production' || process.env.FORCE_BUILD_TESTS;
 }
+
+function minification(config: UserConfig, mode: string) {
+  if (mode !== 'production') {
+    return;
+  }
+
+  /**
+   * Outside of test, the only other time "build" is used,
+   * is production
+   */
+  config.build ||= {};
+
+  if (!config.build.minify) {
+    config.build.minify = 'terser';
+  }
```

Comment:
Rather, should this be checking that `config.build.minify` is undefined? Since it is allowed to be a boolean, if someone sets this `minify: false` in their config (for whatever reason), then this will ignore that

https://vite.dev/config/build-options#build-minify

Reply:
That is a good point -- this should be updated to check undefined specifically